tes_client/communication/request_sender.py

```diff
@@ -247,11 +247,35 @@ def cancel_order(self,
         self._queue_message(tes_message)
         return cancel_order
 
+    def cancel_all_orders(self,
+                          request_header: RequestHeader,
+                          account_info: AccountInfo,
+                          symbol: str = None,
+                          side: str = None):
+        """
+        Sends a request to omega to cancel all orders. Optionally including
+        side and/or symbol
+        :param request_header: Header parameter object for requests.
+        :param account_info: (AccountInfo) Account on which to cancel order.
+        :param symbol: str (optional)
+        :param side: str (optional)
+        :return: (capnp._DynamicStructBuilder) cancel_all_orders object.
+        """
+        tes_message, cancel_all_orders = cancel_all_orders_capnp(
+            request_header=request_header,
+            account_info=account_info,
+            symbol=symbol,
+            side=side)
+        logger.debug('Cancelling All Orders.', extra={'symbol': symbol,
+                                                      'side': side})
+        self._queue_message(tes_message)
```

Comment:
Should we change `tes_message` to something like `capnp_message` then?

Reply:
I'm going to wait on that until we have a more stable version of opt-ex running, since the side effects of renaming methods, classes will require models_common to be updated as well.

Reply:
Will update in another PR.